ArcadiaHook.cs

```diff
@@ -204,8 +134,16 @@ public class ArcadiaHook : Node
             }
         }
 
-        public void AddEditorHook(string type, string field){
-            IFn fn = (IFn)Util.Invoke(RT.var("arcadia.internal.namespace", "eval-ifn"), field);
+        public void AddEditorHook(string type, string field) {
+            // eval-ifn break ios build
+            // IFn fn = (IFn)Util.Invoke(RT.var("arcadia.internal.namespace", "eval-ifn"), field);
+            if (field.Trim().Length() == 0) return;
+
+            String[] parts = field.Split("/");
+            String ns = parts[0].Replace("#'", "");
+            String name = parts[1];
+            RT.load(ns.Replace(".", "/"));
+            IFn fn = (IFn)RT.var(ns, name).getRawRoot();
```

Comment:
fields do not evaluated anymore, so not always backward compatible with old game sources, also I'm not sure about  
`
IFn fn = (IFn)RT.var(ns, name).getRawRoot();
`
not expert in c# or clojure, will it work as before(just suspect it's like getting deref instead of reference and influence how repl working, but I'm not sure)